specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-10-01/mfe.json

```diff
@@ -10964,6 +11161,34 @@
       "x-ms-discriminator-value": "Custom",
       "additionalProperties": false
     },
+    "CustomMetricThreshold": {
+      "required": [
+        "metric"
+      ],
+      "type": "object",
+      "properties": {
+        "metric": {
+          "description": "[Required] The user-defined metric to calculate.",
+          "minLength": 1,
+          "pattern": "[a-zA-Z0-9_]",
```

Comment:
>"pattern": "[a-zA-Z0-9_]", [](http://example.com/codeflow?start=10&length=26)

This pattern means the metric name can start with even digits or "_". Just highlighting in case not intended.

Reply:
Yes that's as expected!